File: 169-majority-element/169-majority-element.cpp
class Solution {
public:
    int majorityElement(vector<int>& nums) {
        int count = 0, element = nums[0];
        for(int i = 0; i < nums.size(); ++i) {
            if(count == 0) element = nums[i];
            if(nums[i] == element) count++;
            else count--;
        }
        return element;
    }
};

File: 169-majority-element/169-majority-element.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        count, element = 0, nums[0]
        for i in range (len(nums)):
            if count == 0: 
                element = nums[i]
            if nums[i] == element:
                count += 1
            else: count -= 1
        return element